Geometry/TrackerGeometryBuilder/src/TrackerGeometry.cc

```diff
@@ -247,9 +247,9 @@ void TrackerGeometry::fillTestMap(const GeometricDet* gd) {
 TrackerGeometry::ModuleType TrackerGeometry::getDetectorType(DetId detid) const {
   for (auto iVal : theDetTypetList) {
     DetId detid_max = std::get<0>(iVal);
-    TrackerGeometry::ModuleType mtype = std::get<1>(iVal);
     if (detid.rawId() <= detid_max.rawId())
-      return mtype;
+      return std::get<1>(iVal);
+    // don't get unless necessary...  idr 12/6/19
```

Comment:
please remove the comment,  and pass the message that one should document what the code doea, not what changes have been made to it

Reply:
fixed.